src/screens/helping/services.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import { connectModule } from 'redux-modules'
+import { compose, withStateHandlers } from 'recompose'
+import servicesModule from '../../modules/services'
+import Layout from '../../components/Layout'
+import { Container, HeaderContainer } from '../../components/atoms'
+import { Input, Radio, Checkbox, Button } from 'antd'
+import styled from 'styled-components'
+const RadioGroup = Radio.Group
+const CheckboxGroup = Checkbox.Group
+const { TextArea } = Input
+
+const RequiredIndicator = styled.em`
+  color: red;
+`
+
+const Label = styled.div`
+  font-size: 18px;
+  text-transform: uppercase;
+  padding: 15px 0;
+`
+
+const StackContainer = styled.div`
+  margin: 0 0;
+`
+
+const FormSection = styled.div`
+  padding: 30px 0;
+`
+
+const contactOptions = ['Phone', 'Text Message', 'Email', ]
+
+const StackInput = ({ required, children, label }) => (
+  <StackContainer>
+    <Label>
+      {required && <RequiredIndicator>*</RequiredIndicator>} {label}
+    </Label>
+    <div>
+      {children}
+    </div>
+  </StackContainer>
+)
+
+const Services = ({ actions, update, formData, history }) => (
+  <Layout header="Services" onBack={history.goBack}>
+    <Container style={{ maxWidth: '600px', display: 'flex' }}>
+      <HeaderContainer>
+        Service Information
+      </HeaderContainer>
+      
+      <FormSection>
+        <StackInput required label="Your Name:">
+          <Input
+            onChange={({ target }) => update('name', target.value)}
+          />
+        </StackInput>
+
+        <StackInput required label="Organization/Company:">
+          <Input
+            onChange={({ target }) => update('update-this-field', target.value)}
+          />
+        </StackInput>
+
+        <StackInput required label="Phone Number:">
+          <Input
+            onChange={({ target }) => update('phoneNumber', target.value)}
+          />
+        </StackInput>
+
+        <StackInput required label="Email Address:">
+          <Input
+            onChange={({ target }) => update('emailAddress', target.value)}
+          />
+        </StackInput>
+
+        <StackInput label="Website:">
+          <Input
+            onChange={({ target }) => update('update-this-field', target.value)}
+          />
+        </StackInput>
+      </FormSection>
+
+      <FormSection>
+        <StackInput required label="Service Fees:">
+          <RadioGroup value={formData.duration} onChange={({ target }) => update('duration', target.value)}>
```

Comment:
The `formData` variable should be modified to work for the new form you have here. 

Since the backend isn't in place yet can make guesses about what the correct key should be. In this case `formData.serviceFees` would make sense.

Once the backend is implemented later on, we just need to map the keys we made up on the frontend to what the backend expects. The module has logic for this already.

Reply:
Aye aye captain